src/components/expression/expression-renderer.js

```diff
@@ -551,6 +713,9 @@ genotet.ExpressionRenderer.prototype.drawMatrixCells_ = function() {
   var heatmapRects = heatmapRows.selectAll('rect').data(_.identity);
   heatmapRects.enter().append('rect');
   heatmapRects
+    .attr('class', function(value, i, j) {
+      return 'cell row' + j + ' column' + i;
```

Comment:
There can be thousands of rects and writing "cell" class to each one costs at least thousands of extra characters in DOM. Why not giving drag rect a special class, and then use a css not selector to define styles for all other rects.